ads/adyoulike.md

```diff
@@ -0,0 +1,43 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Adyoulike
+
+Serves ads from [Adyoulike](https://www.adyoulike.com/).
+
+## Example
+
+### Normal ad
+
+```html
+<amp-ad width="300" height="250"
+      type="adyoulike"
+      data-placement=""
```

Comment:
Could you explain why this is required if it is an empty value?

Reply:
Actually it seems as this is only for this example, but is a bit confusing. Can you put a fake value here?

Reply:
Yes it's only for this example. I put a value

Reply:
Thanks!